scenario/BulkerScenario.ts

```diff
@@ -165,6 +165,7 @@ scenario(
     filter: async (ctx) => await isBulkerSupported(ctx) && await isRewardSupported(ctx) && !matchesDeployment(ctx, [{deployment: 'weth'}, { network: 'linea-goerli' }]),
     supplyCaps: {
       $asset0: 3000,
+      $asset1: 3000,
```

Comment:
In this action, the scenario does operation with asset0, which is OP, and asset1, which is WETH. The scenario would fail because WETH does not have supply caps in the configuration as it is a deployment PR.